cli/fs.rs

```diff
@@ -108,3 +114,19 @@ pub fn normalize_path(path: &Path) -> String {
     s
   }
 }
+
+#[cfg(unix)]
+pub fn chown(path: &str, uid: u32, gid: u32) -> DenoResult<()> {
+  use crate::errors::DenoError;
+  let nix_uid = Uid::from_raw(uid);
+  let nix_gid = Gid::from_raw(gid);
+  unix_chown(path, Option::Some(nix_uid), Option::Some(nix_gid))
+    .map_err(DenoError::from)
+}
+
+#[cfg(not(unix))]
+pub fn chown(_path: &str, _uid: u32, _gid: u32) -> DenoResult<()> {
+  // Noop
+  // TODO: implement chown for Windows
+  errors::op_not_implemented()
```

Comment:
I think you need to wrap this with Err()

Reply:
Yes, sorry about that! I'm on a Mac and just realized the Windows implementation has not been tested locally...